src/gc/env/gcenv.ee.h

```diff
@@ -78,7 +80,7 @@ class GCToEEInterface
 
     static void GcEnumAllocContexts(enum_alloc_context_func* fn, void* param);
 
-    static void AttachCurrentThread(); // does not acquire thread store lock
+    static bool CreateBackgroundThread(Thread** thread, GCBackgroundThreadFunction threadStart, void* arg);
```

Comment:
Hmm, but thinking about it, it seems it would be nicer to modify the pre-existing code for redhawk to actually wait for the thread to launch inside the CreateBackgroundThread. And then we can return the Thread*.